### PR TITLE
New: Added Tranform compatibility to advanced heading and blockquote block, removed from icon list block

### PR DESCRIPTION
**Pull request by supriyajjadhav**
[Trello Task](https://trello.com/c/aeyrOzbo/50-transform-compatibility)
### Description
Block Transforms is the API that allows a block to be transformed from and to other blocks, as well as from other entities. 

### Types of changes
Added transform compatibility in the Advanced heading ,blockquote module.
Removed Transform Compatibility in icon list block

### How has this been tested?
Tested whether it is working on the editor

### Checklist:
- [x] My code is tested
- [x] My code follows accessibility standards 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Comment by AbhijitNage123** [comment]
@supriyajjadhav Please add a changelog for it.

**Comment by AbhijitNage123** [comment]
@supriyajjadhav Please remove conflict. checkout readme file

**Review by AbhijitNage123 — APPROVED** [review]

**Review by RajkiranBagal — COMMENTED** [review]
@supriyajjadhav Update changelog, & add proper PR title ie. To which blocks we have added transform compatibility & which we removed. Something like that.

**Review by RajkiranBagal — APPROVED** [review]

File: readme.txt [hunk 1]
```diff
@@ -165,6 +165,9 @@ The awesome React FontIconPicker is developed by [Alessandro Benoit](http://code
 1. /assets/screenshots/1.png
 
 == Changelog ==
+= 1.21.0
+* New: Transform Compatibility added in Advanced Heading and Blockquote Block.
```
Comment (vrundakansara) [inline readme.txt:169]:
```txt
* Improvement: Added Transform Compatibility in Advanced Heading and Blockquote Block.
* Deprecated: Removed Transform Compatibility from Icon List Block.
```

**Review by vrundakansara — APPROVED** [review]

**Pull request merged**